lexers/testdata/markdown.expected

```diff
@@ -1,427 +1,166 @@
-  {"type":"Name","value":"r"},
-  {"type":"Text","value":" "},
-  {"type":"NameBuiltin","value":"record"},
-  {"type":"Punctuation","value":";"},
-  {"type":"Text","value":"\n"},
-  {"type":"Keyword","value":"begin"},
-  {"type":"Text","value":"\n  "},
-  {"type":"Keyword","value":"for"},
-  {"type":"Text","value":" "},
-  {"type":"Name","value":"r"},
-  {"type":"Text","value":" "},
-  {"type":"Keyword","value":"in"},
-  {"type":"Text","value":" "},
-  {"type":"Keyword","value":"select"},
-  {"type":"Text","value":" "},
-  {"type":"Operator","value":"*"},
-  {"type":"Text","value":" "},
-  {"type":"Keyword","value":"from"},
-  {"type":"Text","value":" "},
-  {"type":"Name","value":"books"},
-  {"type":"Text","value":"\n"},
-  {"type":"Keyword","value":"loop"},
-  {"type":"Text","value":"\n  "},
-  {"type":"Keyword","value":"execute"},
-  {"type":"Text","value":" "},
-  {"type":"LiteralStringSingle","value":"'select '''"},
-  {"type":"Text","value":" "},
-  {"type":"Operator","value":"||"},
-  {"type":"Text","value":" "},
-  {"type":"Name","value":"r"},
-  {"type":"LiteralNumberFloat","value":"."},
-  {"type":"Name","value":"title"},
-  {"type":"Text","value":" "},
-  {"type":"Operator","value":"||"},
-  {"type":"Text","value":" "},
-  {"type":"LiteralStringSingle","value":"''''"},
-  {"type":"Punctuation","value":";"},
-  {"type":"Text","value":"\n"},
-  {"type":"Keyword","value":"end"},
-  {"type":"Text","value":" "},
-  {"type":"Keyword","value":"loop"},
-  {"type":"Punctuation","value":";"},
-  {"type":"Text","value":"\n"},
-  {"type":"Keyword","value":"end"},
-  {"type":"Text","value":"\n"},
-  {"type":"LiteralStringHeredoc","value":"$$"},
-  {"type":"Punctuation","value":";"},
-  {"type":"Text","value":"\n\n"},
-  {"type":"Keyword","value":"DO"},
-  {"type":"Text","value":" "},
-  {"type":"LiteralStringHeredoc","value":"$$\ndeclare r record;\nbegin\n  for r in select * from books\nloop\n  execute 'select ''' || r.title || '''';\nend loop;\nend\n$$"},
-  {"type":"Punctuation","value":";"},
-  {"type":"Text","value":"\n"},
-  {"type":"LiteralString","value":"```"},
+  {"type":"GenericHeading","value":"# about\r\n"},
+  {"type":"Text","value":"\r"},
+  {"type":"Error","value":"\n"},
+  {"type":"GenericSubheading","value":"## user defined function in cql\r\n"},
+  {"type":"Text","value":"\r"},
+  {"type":"Error","value":"\n"},
+  {"type":"Text","value":"```javascript\r"},
+  {"type":"Error","value":"\n"},
+  {"type":"Text","value":"  column.substring(0,num) \r"},
+  {"type":"Error","value":"\n"},
+  {"type":"Text","value":"```\r"},
+  {"type":"Error","value":"\n"},
+  {"type":"Text","value":"\r"},
+  {"type":"Error","value":"\n"},
+  {"type":"Text","value":"```cql\r"},
+  {"type":"Error","value":"\n"},
+  {"type":"Text","value":"CREATE FUNCTION IF NOT EXISTS cycling.left (column TEXT,num int)\r"},
+  {"type":"Error","value":"\n"},
+  {"type":"Text","value":"RETURNS NULL ON NULL INPUT\r"},
+  {"type":"Error","value":"\n"},
```

Comment:
`\n` shouldn't be an error. Something in the regex change has introduced this.

Reply:
I agree. But I cannot find the error in my changes. If I generate a Markdown test file on the `master` branch, I also end up with 'Error' values in `markdown.expected`. 

That would suggest the error cause was already there before I worked on this PR. Do I understand that correctly?

Reply:
It's possible it was already there, but it's certainly much worse now...

Reply:
I get 79 'Error' values in `markdown.expected` on `master` and there are also 79 'Error' values in `markdown.expected` in this pull request. So on my side the amount of errors are the same.

Is this case perhaps similar to the line endings difference we discussed [earlier](https://github.com/alecthomas/chroma/pull/172#issuecomment-422238512)?

Reply:
There are 4 on master:

```
$ grep Error markdown.expected | wc -l
       5
```

Reply:
But yeah, it does look like the same issue.

Reply:
Yes there are 5 on the `master` file. But the problem I have is that when I make a *new* test file on `master`, there are plenty more errors. Can you check those on your end?

Because no matter what I test (on `master`), I always get significantly more errors than the 5 currently committed to master. So either something with Chroma changed that made it render different test files than in June. Or testing works differently on Windows than Linux.

Reply:
I did and there are only 5.

Reply:
Thanks for looking into it, much appreciated. :+1: 

I'll close this pull request since today I neither succeeded in getting clean testdata from the Markdown lexer on `master`.